Mathlib/Data/Quot.lean

```diff
@@ -46,6 +46,8 @@ instance (r : α → α → Prop) [Inhabited α] : Inhabited (Quot r) :=
 protected instance Subsingleton [Subsingleton α] : Subsingleton (Quot ra) :=
   ⟨fun x ↦ Quot.induction_on x fun _ ↦ Quot.ind fun _ ↦ congr_arg _ (Subsingleton.elim _ _)⟩
 
+@[deprecated (since := "2024-08-26")] alias recOnSubsingleton' := Quot.recOnSubsingleton
```

Comment:
```suggestion
@[deprecated (since := "2024-08-26")] alias recOnSubsingleton' := Quot.recOnSubsingleton

@[deprecated (since := "2024-08-26")] alias recOn' := Quot.recOn
```